pkgs/misc/emulators/duckstation/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, mkDerivation, fetchFromGitHub, fetchpatch, cmake, pkgconfig, SDL2, qtbase
+, wrapQtAppsHook, qttools, ninja, git, gtk3, glib }:
```

It seems `fetchpatch `, `git` and `glib` and not needed anymore

`pkg-config` is spelled with `-` (and `pkgconfig` is an alias)

```suggestion
{ lib, mkDerivation, fetchFromGitHub, cmake, pkg-config, SDL2, qtbase
, wrapQtAppsHook, qttools, ninja, gtk3 }:
```